docs/push/development.md

```diff
@@ -0,0 +1,166 @@
+---
+id: development
+title: Development Process
+---
+
+We develop and deploy on a 2 week schedule. A new release is
+tagged and cut a week before deploy, QA files a deployment ticket filed then verifies the release. On occasion a release
+will get a full suite of load-tests as well.
+
+Some of the ancillary Push projects have no set release schedule and are
+released as needed.
+
+## Code Development
+
+Each Github feature card being actively developed has a milestone created in the main
+[Mozilla Push-Service Repository][MPSM], which is then propagated out to all
```

Comment:
Since this is for `application-services` should we point to that repo?

Reply:
Ah, I thought I removed this actually.